src/TypeScript/Analyzer/TypeScriptAnalyzer.cs

```diff
@@ -105,15 +97,7 @@ internal async Task ExecuteAnalysis(string filePath, IIssueConsumer consumer, Ca
                 }
 
                 logger.WriteLine("[TypescriptAnalyzer] time to find ts config: " + stopwatch.ElapsedMilliseconds);
-
-                stopwatch.Restart();
-                var issues = await eslintBridgeAnalyzer.Analyze(filePath, tsConfig, cancellationToken);
-                analysisStatusNotifier.AnalysisFinished(issues.Count, stopwatch.Elapsed);
-
-                if (issues.Any())
-                {
-                    consumer.Accept(filePath, issues);
-                }
+                stopwatch.Stop();
             }
             catch (TaskCanceledException)
```

Comment:
I think you could make a virtual method GetTsConfig which would be empty (return null) in the base and overridden in the TS analyzer. This would allow to move the invocation inside the try catch in the AnalysisBase and get rid of this try catch